Docs/changelog/2024 02 06 145913 (#2699)
scaleway/docs-content: changelog/february2024/2024-02-06-kubernetes-security-runc-bumped-to-1112-for-cve-2024-21.mdx

---
title: Runc bumped to 1.1.12 for CVE-2024-21626
status: security
author:
    fullname: 'Join the #k8s channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-02-06
category: containers
product: kubernetes
---

[CVE-2024-21626](https://snyk.io/blog/leaky-vessels-docker-runc-container-breakout-vulnerabilities/) is a recently disclosed security issue affecting the `runc` component.

> To apply the patch, managed Kubernetes services customers are recommended to take the following actions:

Reviewer: ```suggestion
To apply the patch, managed Kubernetes services customers should take the following actions:
```
